### PR TITLE
Directory is empty then load_all function should return empty list

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bug-id: Tendrl/commons#850
tendrl-bug-id: Tendrl/gluster-integration#576

Signed-off-by: GowthamShanmugamsundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n please review

File: tendrl/commons/objects/__init__.py [hunk 1]
```diff
@@ -169,8 +169,10 @@ def load_all(self):
                         return None
         ins = []
         for item in etcd_resp.leaves:
-            self.value = item.key
-            ins.append(self.load())
+            if item.key.strip("/") != value.strip("/"):
```
Comment (shtripat) [inline tendrl/commons/objects/__init__.py]: what is this checking of key and value part? Can you add an example here?

Comment (GowthamShanmugam) [reply tendrl/commons/objects/__init__.py]: i will add

**Comment by GowthamShanmugam** [comment]
Tested

**Pull request merged**